src/main/java/paymelah/logic/parser/ParserUtil.java

```diff
@@ -214,4 +215,12 @@ public static DebtContainsKeywordsPredicate prepareDebtContainsKeywordsPredicate
     public static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
         return Stream.of(prefixes).allMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
     }
+
+    /**
+     * Parses {@code String s} into a {@code DebtGreaterEqualAmountPredicate}.
+     */
+    public static DebtGreaterEqualAmountPredicate parseDebtGreaterEqualAmountPredicate(String s) throws ParseException {
```

Comment:
Bryan renamed to prepareXPredicate, should probably rename this to keep consistency.